src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const ButtonSearch = props => {
+  return (
+    <button onClick={props.search} className="btn btn-primary fn-submit-name">
```

Comment:
Good work implementing the click handler 👍

Reply:
Thanks Ali